tests/lib/rules/require-explicit-emits.js

```diff
@@ -1983,6 +1983,25 @@ emits: {'foo': null}
         }
       ],
       ...getTypeScriptFixtureTestOptions()
+    },
+    {
+      filename: 'test.vue',
+      code: `
+      <template>
+        <div @click="emits('bar')"/>
+      </template>
+      <script setup lang="ts">
+      const emits = defineEmits<(e: 'foo') => void>()
```

Comment:
The convention is to call the return value of `defineEmits` `emit`, since it is a function that will emit the given event.

```suggestion
        <div @click="emit('bar')"/>
      </template>
      <script setup lang="ts">
      const emit = defineEmits<(e: 'foo') => void>()
```

Reply:
Thanks for pointing that out!